src/helpers/responseGenerator.js

```diff
@@ -0,0 +1,42 @@
+import utility from './utils';
+
+const responseGenerator = {
+  /**
+   * @param {*} res - express response object
+   * @param {*} statusCode - response status code
+   * @param {*} [message=undefined] - error message
+   * @returns {Object} return a failed response
+   */
+  sendError(res, statusCode, message = undefined) {
+    return res.status(statusCode).send({
+      status: 'error',
+      message
+    });
+  },
+
+  /**
+   * @param {Object} res - express response object
+   * @param {int} statusCode - response status code
+   * @param {*} data - response status code
+   * @param {string} [message=undefined] - error message
```

Comment:
setting message to `undefined` by default is same as not setting a value to start with. A more useful option would be setting it to an empty string by default if it'll always be a `String`